public/JavaScript/gallery.js

```diff
@@ -1,48 +1,34 @@
-// document.addEventListener('DOMContentLoaded', () => {
-//   let selectedAnimalType = null
-//   const galleryItems = document.querySelectorAll('.gallery-items')
-
-//   // Function to hide all gallery items
-//   const hideAllAnimals = (elements) => {
-//     elements.forEach(item => {
-//       item.style.display = 'none'
-//     })
-//   }
-//   // Function to display selected animal
-//   const showSelectedAnimals = () => {
-//     hideAllAnimals(document.querySelectorAll('.gallery-items'))
-//     if (selectedAnimalType) {
-//       galleryItems.forEach(item => {
-//         if (item.dataset.animalType === selectedAnimalType) {
-//           item.style.display = 'block'
-//         }
-//       })
-//     }
-//   }
-//   // Target animal type
-//   function setSelectedAnimalType (type) {
-//     const filterContainer = document.querySelector('#singleAnimalFilter')
-//     if (filterContainer) {
-//       filterContainer.addEventListener('click', (event) => {
-//         const filterOption = event.target.closest('[data-animal-type]')
-//         if (filterOption) {
-//           selectedAnimalType = filterOption.dataset.animalType
-//           console.log(`Selected animal type: ${selectedAnimalType}`)
-//         }
-//       })
-//     } else {
-//       console.error('Filter container not found!')
-//     }
-//   }
-//   // Apply button functionality
-//   const applyFilters = document.querySelector('#apply-filters')
-//   if (applyFilters) {
-//     applyFilters.addEventListener('click', showSelectedAnimals)
-//   } else {
-//     console.error('Apply filters button not found')
-//   }
-//   setSelectedAnimalType()
-// })
+document.addEventListener('DOMContentLoaded', function () {
+  // Event listeners for animal filter buttons
+  document.querySelectorAll('#singleAnimalFilter div[data-type]').forEach(selectingDivFilter => {
+    selectingDivFilter.addEventListener('click', function () {
```

Comment:
userSelectedFilters, activeFilters, galleryItems are great. 

Names such as selectingDivFilter not so much. What is a div? If I change the html from a div to a span will this name still make sense?